core/src/test/resources/org/elasticsearch/index/query/not-filter3.json

```diff
@@ -1,7 +0,0 @@
-{
-            "not":{
-                "term":{
-                    "name.first":"shay1"
```

Comment:
Just curious where this was moved since it was the special test case that tripped when I was trying to remove the by-name parsing in NotQueryParser.